apps/block_scout_web/lib/block_scout_web/schema/types.ex

```diff
@@ -62,6 +62,8 @@ defmodule BlockScoutWeb.Schema.Types do
     field(:account_type, :string)
     field(:nonvoting_locked_gold, :wei)
     field(:locked_gold, :wei)
+    field(:attestations_requested, :integer)
```

Comment:
Is this supposed to be in hree?

Reply:
Yeah, probably should be moved once the endpoints for validators and validator groups have been implemented.